compiler/circle2circle-dredd-recipe-test/test.lst

```diff
@@ -96,6 +96,7 @@ Add(PadV2_001 PASS substitute_padv2_to_pad)
 Add(Softmax_001 PASS decompose_softmax)
 Add(Softmax_002 PASS decompose_softmax)
 Add(StridedSlice_003 PASS substitute_strided_slice_to_reshape)
+Add(Inf_Pad_000 PASS)
```

Comment:
Please sort in alphabetical order

Reply:
> Please sort in alphabetical order

Instead of sorting, what about separate `Inf_` tests? 

```suggestion
# Shape Inference test
Add(Inf_Pad_000 PASS)
```

Reply:
> , what about separate `Inf_` tests?

## 👍